tests/dummy/app/pods/demo/template.hbs

```diff
@@ -40,13 +40,45 @@ frost-modal testbed
       <div>Test Information block</div>
     </div>
   {{/block-slot}}
-  {{#block-slot slot 'footer' as |controls|}}
+  {{#block-slot slot 'footer' as |controls close|}}
     {{controls.cancel}}
     {{controls.button
       text='Custom action'
       onClick=(action 'myCustomAction')}}
     {{controls.confirm
       onConfirm=(action 'confirmHandler')
-      text='Confirm it!'}}
+      text='Confirm and close'}}
+  {{/block-slot}}
+{{/frost-modal}}
+
+<br>
+Non default dialog
+{{#frost-modal
+  modalName='testModal2'
+  closeOnClick=false as |slot|}}
```

Comment:
I think this belongs on the confirmation control, not on the `frost-modal` interface